src/js/burger-menu.js

```diff
@@ -23,19 +25,14 @@ document.addEventListener('DOMContentLoaded', () => {
 
   navLinks.forEach((link) => {
     link.addEventListener('click', (e) => {
+      route(e);
```

Comment:
Please pass the route function as a callback to the event listener, you don't need to call it inside callback, just pass only it

Reply:
fixed